Cargo.toml

```diff
@@ -28,10 +36,13 @@ repository = "https://github.com/shuttle-hq/shuttle"
 
 # https://doc.rust-lang.org/cargo/reference/workspaces.html#the-workspacedependencies-table
 [workspace.dependencies]
-shuttle-codegen = { path = "codegen", version = "0.20.0" }
-shuttle-common = { path = "common", version = "0.20.0" }
-shuttle-proto = { path = "proto", version = "0.20.0" }
-shuttle-service = { path = "service", version = "0.20.0" }
+# libraries
+shuttle-codegen = { path = "codegen" }
+shuttle-common = { path = "common" }
+shuttle-proto = { path = "proto" }
+shuttle-service = { path = "service" }
```

Comment:
@oddgrd Do you think removing the versions here will break something in the release process? I'm down to just "if it ain't broke, don't fix it" and add them back.

Reply:
Yes, I think we would need the versions to be able to publish the crates that use these. You should be able to test this with [`cargo publish --dry-run`](https://doc.rust-lang.org/cargo/commands/cargo-publish.html#publish-options) in one of the crates.